conans/test/unittests/client/tools/test_version.py

```diff
@@ -47,6 +47,12 @@ def test_valid_loose(self):
         self.assertEqual(v.minor, "0")
         self.assertEqual(v.patch, "0")
 
+        v = Version("1.2.3.4")
+        self.assertEqual(v.major, "1")
+        self.assertEqual(v.minor, "2")
+        self.assertEqual(v.patch, "3")
+        self.assertEqual(v.micro_versions, "4")
```

Comment:
Can micro be more than one? If yes, then add more in testing too.
I think it would be good to call it just "micro" not "micro_versions", why not?

Reply:
Yeah, I prefer micro too. And yes, it can be a list with more than one number. I'll extend this test

Reply:
Done